### PR TITLE
bbbbb

### PR DESCRIPTION
**Pull request by barosl2**
THIS IS THE DESCRIPTION

AND MORE

### CODE REVIEW

File: b.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+b
```
Comment (barosl2) [inline b.txt:1]: @barosl3 r+

Comment (barosl2) [inline b.txt:1]: abc

Comment (barosl2) [inline b.txt:1]: def

Comment (barosl2) [inline b.txt:1]: @barosl3 r+

**Comment by barosl3** [comment]
:zap: Test exempted - [status](https://travis-ci.org/barosl2/test4/builds/93443677)

File: b.txt (hunk 1 above)
Comment (barosl2) [inline b.txt:1]: AND AGAIN

Comment (barosl2) [inline b.txt:1]: 깔깔

Comment (barosl2) [inline b.txt:1]: 호호

**Pull request merged**